roles/nova-data/templates/cacert.key.j2

```diff
@@ -0,0 +1 @@
+{{ libvirt_ssl.key }}
```

Comment:
PR is in openstack-envs.  this reminds me that i need to add some variables to the bundled envs in this repo.